gpfit/tests/t_initialize.py

```diff
@@ -3,7 +3,7 @@
 import numpy as np
 from numpy import arange, newaxis, vstack, log, exp
 from numpy.random import random_sample
-from gpfit.initialize import get_initial_parameters
+from gpfit.maths.initialize import get_initial_parameters
```

Comment:
great job sneaking in the plural `maths` :) approved.

Reply:
Haha thanks thought you might enjoy